build.gradle

```diff
@@ -63,12 +63,9 @@ allprojects {
         google()
         jcenter() {
             content {
-                includeModule('de.adorsys.android', 'securestoragelibrary')
-                includeModule('com.andrognito.pinlockview', 'pinlockview')
-                includeModule('me.toptas.fancyshowcase', 'fancyshowcaseview')
-                includeModule('org.matomo.sdk', 'tracker')
-                includeModule('com.journeyapps', 'zxing-android-embedded')
-                includeGroup('co.infinum')
+                includeModule('de.adorsys.android', 'securestoragelibrary') //Only available version in jitpack is 0.0.2. We use 1.2.2
+                includeModule('com.journeyapps', 'zxing-android-embedded')  //Only works for API 24+ using Maven Repository, 19+ can be used but we have to downgrade zxing to 3.3.0
```

Comment:
we can't downgrade zxing. It is needed for GS1 qr codes

Reply:
That's why we are still using jcenter for this dependency. it can't be remove from that repository just yet